api/PaymentCurrencyAmount.json

```diff
@@ -0,0 +1,344 @@
+{
+  "api": {
+    "PaymentCurrencyAmount": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PaymentCurrencyAmount",
+        "support": {
+          "chrome": {
+            "version_added": "53",
```

Comment:
The flag was removed in Chrome 56 per [this commit](https://storage.googleapis.com/chromium-find-releases-static/18d.html#18d3734940ccbc96eff5246e2d6235145d44a59e). Please change. Notice that the [change list](https://chromium.googlesource.com/chromium/src/+/18d3734940ccbc96eff5246e2d6235145d44a59e) contains several other removals of this flag.

Reply:
Hopefully all is well now...